src/anm/animation/element.js

```diff
@@ -1211,11 +1245,12 @@ Element.prototype.removeModifier = function(modifier) {
     if (!this.__modifiers_hash[modifier.id]) throw new AnimationError('Modifier wasn\'t applied to this element');
     if (!modifier.__applied_to || !modifier.__applied_to[this.id]) throw new AnimationError(Errors.A.MODIFIER_NOT_ATTACHED);
     //if (this.__modifying) throw new AnimErr("Can't remove modifiers while modifying");
-    delete this.__modifiers_hash[modifier.id];
-    delete this.$modifiers[modifier.type].splice(modifier.__applied_to[this.id] - 1, 1); // delete by index
-    delete modifier.__applied_to[this.id];
+    utils.removeElement(this.__modifiers_hash, modifier.id);
```

Comment:
Second thought: `disposeValue` is also not a "true", I only do it to:

* some algorithms are based on property-check here and there, so if, for example, the modifier ID is not in modifiers hash _object_, modifier is treated as removed; I believe these algorithms will break in given case;
* break references so if user has no pointers to it, it won't take the memory;